cognite/neat/graph/stores/_graphdb_store.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+import requests
+from rdflib import Graph, Namespace
+from rdflib.plugins.stores.sparqlstore import SPARQLUpdateStore
+
+from cognite.neat.constants import DEFAULT_NAMESPACE, PREFIXES
+
+from ._base import NeatGraphStoreBase
+
+
+class GraphDBStore(NeatGraphStoreBase):
+    """GraphDB is a class that stores the graph in a GraphDB instances and provides methods to
+    read/write data it contains
+
+
+    Args:
+        graph : Instance of rdflib.Graph class for graph storage
+        base_prefix : Used as a base prefix for graph namespace, allowing querying graph data using a shortform of a URI
+        namespace : Namespace (aka URI) used to resolve any relative URI in the graph
+        prefixes : Dictionary of additional prefixes used in the graph
+    """
+
+    rdf_store_type = "graphdb"
```

Comment:
Should this be enum / StrEnum?

Reply:
No, I do not think so, as that would be causing unnecessary coupling, i.e., adding a new typo of store would cause you to not only create a new class inheriting from `NeatGraphStoreBase` but you would also have to then update that enum.